src/features/unified-share-modal/UnifiedShareModal.scss

```diff
@@ -92,6 +92,7 @@
     }
 
     .bdl-PillSelectorDropdown .bdl-PillSelector, // need both selectors for specificity
+    .pill-selector-wrapper .pill-selector-input-wrapper,
```

Comment:
Out of curiosity, why are these being added back now? Haven't we already done the work to use the new classNames in the parent app where needed?

Reply:
Good point. I will remove these.